src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/arm/convert.cpp

```diff
@@ -0,0 +1,169 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/convert.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+namespace ov {
+namespace test {
+namespace snippets {
+
+
+namespace {
+
+const std::vector<std::pair<std::vector<ov::element::Type>, std::vector<ov::element::Type>>> types_Convert = {
```

Comment:
It can be minor, if used functions are covered in other conversions. For example: test case is not covered: store, `i32` => `i8` / `u8`, but function `cvt_i32_to_byte` is used in other cases (actually, I'm not sure about all function input arguments). Please, be sure that used functions are covered. 

Not covered conversions:
* store: `i32` => `i8` / `u8`
* store: `i32` => `f16`
* store: `f32` => `i32`
* store: `i32` => `f32`
* load: `i8` / `u8` => `i32`
* load: `f16` => `i32`
* load: `i32` => `f32`
* load: `f32` => `i32`

Reply:
Thanks Edward for the comment! In fact, I added these i32 related test cases during the implementation. Then I removed them to align the tokenization behavior with x64, described here https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp#L1043.